working-groups/query-standardization/dsl-survey/kusto-query-language-metrics/README.md

```diff
@@ -0,0 +1,238 @@
+# Kusto Query Language, Metrics Variant DSL Survey Response
+
+### **Short Description:**
+
+The Kusto Query Language (KQL-M) DSL is a variant of the Kusto Query Language (KQL) DSL that used in
```

Comment:
KQL-M stands for 'Kusto Query Language for Metric". Would suggest to modify text accordingly, i.e.:
The Kusto Query Language for Metrics (KQL-M) DSL is a variant...